village/src/components/SmurfForm.js

```diff
@@ -13,7 +14,17 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
-
+    const smurf = {
+      name:this.state.name,
+      age:this.state.age,
+      height:this.state.height
+    }
+    axios
+      .post('http://localhost:3333/smurfs', smurf)
+      .then(response =>{
```

Comment:
What happens with your response!? Should you handle something that's being returned to you?